RealmSwift/Results.swift

```diff
@@ -147,6 +147,84 @@ extension Projection: KeypathSortable {}
         return RLMIterator(collection: collection)
     }
 
+    // MARK: Flexible Sync
+
+#if swift(>=5.8)
+    /**
+     Creates a SyncSubscription matching the Results' local filter.
+
+     After committing the subscription to the realm's local subscription set, the method
```

Comment:
The docs for this function are a mess when rendered by docc. Note that Xcode 15 now has an assistant editor for previewing rendered docs so you can see them side-by-side with the source rather than having to build the docs every time.